### PR TITLE
Feat: Support sensors for external tables in Airflow

### PR DESCRIPTION
**Pull request by izeigerman**
Docs will be updated in a follow-up PR.

### CODE REVIEW

File: sqlmesh/schedulers/airflow/operators/hwm_sensor.py [hunk 1]
```diff
@@ -63,3 +63,33 @@ def _compute_target_high_water_mark(
...
+        )
+        self.snapshot = snapshot
+        self.external_table_sensor_factory = external_table_sensor_factory
+
+    def poke(self, context: Context) -> t.Union[bool, PokeReturnValue]:
+        dag_run = context["dag_run"]
+        signals = self.snapshot.model.render_signals(
+            start=to_datetime(dag_run.data_interval_start),
+            end=to_datetime(dag_run.data_interval_end),
+            execution_time=now(),
```
Comment (tobymao) [inline sqlmesh/schedulers/airflow/operators/hwm_sensor.py:92]: now or now_timestamp?

**Review by tobymao — APPROVED** [review]

File: sqlmesh/schedulers/airflow/operators/hwm_sensor.py (hunk 1 above)
Comment (izeigerman) [reply sqlmesh/schedulers/airflow/operators/hwm_sensor.py:92]: All other arguments are datetime instances, so I figured I should be consistent.

Comment (tobymao) [reply sqlmesh/schedulers/airflow/operators/hwm_sensor.py:92]: that's fine, just understand that it's minute floored

Comment (izeigerman) [reply sqlmesh/schedulers/airflow/operators/hwm_sensor.py:92]: oh, nice gotcha 😢

**Pull request merged**